backend/frontend/public/locales/en/updatepassword.json

```diff
@@ -0,0 +1,17 @@
+{
+  "userProfile": "User Profile",
+    "updatePassword": "Update Password",
+    "home": "Home",
+    "profile": "Profile",
+    "updateProfile": "Update Profile",
+    "changePassword": "Change Password",
+    "newPassword": "New Password",
+    "confirmPassword": "Confirm Password",
+    "enterPassword": "Enter password",
+    "enterPasswordAgain": "Enter password again",
+    "resetPassword": "Reset Password",
+    "passwordUpdated": "Password updated successfully",
+    "passwordsDoNotMatch": "Passwords do not match",
+    "invalidPassword": "Password must contain minimum 6 characters, one special character, and one number"
```

Comment:
Do we need both comma (,) and `and` here.

Reply:
Yes, comma is unnecessary.